contracts/dev/rubic.sol.txt

```diff
@@ -0,0 +1,2433 @@
+// IGNORE_LICENSE-Identifier: MIT
```

Comment:
Hi @YoungHS-NUS , I think we should remove the `.txt` extension. I uploaded the file in the original issue as txt only because github rejects files with `.sol` extensions.